### PR TITLE
Adding new relationship detection to remote template rule

### PR DESCRIPTION
**Pull request by aidenmitchell**
As a result of https://delivr.to/payloads?id=ef187d5d-3188-483e-b3b5-9ab5e0e032f7.

### CODE REVIEW

**Comment by morriscode** [comment]
💪 
`regex.icontains(.target, "html?!", "dotm"))`

Since we're looking for dotm, should we look for other macro enabled ole file types too? Thinking about how exhaustive the relationship inspection should be?

**Comment by aidenmitchell** [comment]
> should we look for other macro enabled ole file types too

Yeah that's a good idea, I'll add those. I don't think we can reference the macro extension list, due to the regex, but that's ok

**Comment by morriscode** [comment]
Apologies for delays, retroing now.!

**Review by morriscode — APPROVED** [review]
👍

File: detection-rules/attachment_office_remote_html_template.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ source: |
               // detect files without an extension
               or .file_type in ("docx")
           )
-          and any(file.oletools(.).relationships, regex.icontains(.target, "html?!"))
+          and any(file.oletools(.).relationships, regex.icontains(.target, "html?!", "doc", "docm", "docx", "dot", "dotm", "pptm", "ppsm", "xlm", "xls", "xlsb", "xlsm", "xlt", "xltm"))
```
Comment (rw-access) [inline detection-rules/attachment_office_remote_html_template.yml:36]: what do `htm!` and `html!` mean? looks like the `!` is interpreted literally in RE2, not sure if that's intended.

Comment (rw-access) [reply detection-rules/attachment_office_remote_html_template.yml:36]: maybe @jkamdjou knows since those were in the original rule

Comment (aidenmitchell) [reply detection-rules/attachment_office_remote_html_template.yml:36]: yeah not sure...

**Pull request closed without merging**